substrate/frame/safe-mode/src/lib.rs

```diff
@@ -15,6 +15,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! # Safe Mode
+//!
+//! Trigger for stopping all extrinsics outside of a specific whitelist.
+//!
+//! ## Pallet API
+//!
+//! See the [`pallet`] module for more information about the interfaces this pallet exposes,
+//! including its configuration trait, dispatchables, storage items, events and errors.
+//!
+//! ## Overview
+//!
+//! The safe-mode pallet provides a big STOP button to to put the chain in safe mode and thereby
```

Comment:
Redundant

Reply:
Updated this section some to try to make it more useful.